kani-driver/src/cbmc_output_parser.rs

```diff
@@ -549,31 +550,30 @@ pub fn process_cbmc_output(
     mut process: Child,
     extra_ptr_checks: bool,
     output_format: &OutputFormat,
-    output_filename_opt: Option<&Path>,
-) -> VerificationStatus {
+) -> VerificationOutput {
     let stdout = process.stdout.as_mut().unwrap();
     let mut stdout_reader = BufReader::new(stdout);
-    let parser = Parser::new(&mut stdout_reader, output_filename_opt);
+    let parser = Parser::new(&mut stdout_reader);
     let mut result = false;
-
-    for item in parser {
```

Comment:
Splitting this loop into two (one to process, another to print) means we would not print anything in non-terminating cases (as in #493), unless I'm missing something. We need to keep the original loop here.

Reply:
Ah, thanks for pointing that out. I fixed it.